lib/mission_control/jobs.rb

```diff
@@ -16,5 +16,6 @@ module Jobs
     mattr_accessor :delay_between_bulk_operation_batches, default: 0
     mattr_accessor :logger, default: ActiveSupport::Logger.new(nil)
     mattr_accessor :internal_query_count_limit, default: 500_000 # Hard limit to keep unlimited count queries fast enough
+    mattr_accessor :irb_help, default: true
```

Comment:
Perhaps we could call this `show_console_help` or something like this, to be a bit more explicit? We use `console` in Rails initialization, rather than `irb`.

Reply:
Sure, done